bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/OneWireBindingProvider.java

```diff
@@ -8,38 +8,40 @@
  */
 package org.openhab.binding.onewire;
 
+import java.util.Map;
+
+import org.openhab.binding.onewire.internal.InterfaceAbstractOneWireBindingConfig;
+import org.openhab.core.binding.BindingConfig;
 import org.openhab.core.binding.BindingProvider;
 import org.openhab.core.items.Item;
 
-
-
 /**
- * This interface is implemented by classes that can provide mapping information
- * between openHAB items and OneWire items (sensors).
+ * This interface is implemented by classes that can provide mapping information between openHAB items and OneWire
+ * devices and their properties.
  * 
- * Implementing classes should register themselves as a service in order to be 
- * taken into account.
+ * Implementing classes should register themselves as a service in order to be taken into account.
  * 
- * @author Thomas.Eichstaedt-Engelen
+ * @author Thomas.Eichstaedt-Engelen, Dennis Riegelbauer
  * @since 0.6.0
  */
 public interface OneWireBindingProvider extends BindingProvider {
 
 	/**
-	 * @return the corresponding sensorId to the given <code>itemName</code>
+	 * @return the corresponding InterfaceAbstractOneWireBindingConfig to the given <code>pvItemName</code>
 	 */
-	public String getSensorId(String itemName);
-	
+	public InterfaceAbstractOneWireBindingConfig getBindingConfig(String pvItemName);

```

Comment:
prefixing variable names is not usual in Java. Could you please rename to itemName again?